### PR TITLE
Latest Zcash Release Merge to 1.0.12

### PR DESCRIPTION
**Pull request by csharpee**
merged the latest zcash 1.0.12 release into the hush dev codebase 1.0.9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
 HUSH 1.0.9
+=======
+Zcash 1.0.12
+>>>>>>> b86b9feb6c3c1727cc3992da2f95c4a4f09a982b
```
Comment (leto) [inline README.md]: Looks like you committed some merge conflicts

Comment (csharpee) [reply README.md]: Okay, I think I see what you are talking about. Each of the files that were conflicted I added with git add and then committed. So the files that are in conflict are seem to have a duplication of some code that should not be there.  Tomorrow, I will hopefully correct this. I will open each conflicted file and take a gander at its contents to see the difference between the changes, then correct the conflicts.

**Comment by csharpee** [comment]
Okay @leto I have updated all the conflicting files. The only real problem I seen in the changes is the IPFS integration for Zcash. Does Hush have a IPFS URL we can use? I commented this and left the code there for changes later in case we use IPFS.

File: contrib/debian/changelog [hunk 2]
```diff
@@ -1,134 +1,10 @@
...
- -- Zcash Company <team@z.cash>  Wed, 08 Mar 2017 13:51:11 -0800
-
-zcash (1.0.7) jessie; urgency=medium
+hush (1.0.12) stable; urgency=medium
 
-  * 1.0.7 release.
+  * 1.0.12 release
 
- -- Zcash Company <team@z.cash>  Fri, 03 Mar 2017 20:55:04 -0800
+ -- Hush Team <contact@myhush.org> Wed, 4 Oct 2017 8:08:26 +0800
```
Comment (csharpee) [inline contrib/debian/changelog]: I am not sure if I did the time-stamp correctly.

**Comment by csharpee** [comment]
Okay, I removed the NminerThreads and recompiled. Eureka! We now have a 1.0.12 hush-cli and hushd, well not on GitHub. I have attached an image below for proof. 

image

**Comment by csharpee** [comment]
https://user-images.githubusercontent.com/18726788/31215130-306a4110-a963-11e7-8243-9b2424b708cd.png

**Comment by madbuda** [comment]
PR against https://github.com/MyHush/hush/tree/v1.0.12-lin

**Pull request closed without merging**